code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -257,14 +257,13 @@
 	new /obj/item/reagent_containers/syringe(src)
 	new /obj/item/reagent_containers/glass/bottle/tuberculosiscure(src)
 
-/obj/item/storage/box/syndie_kit/chameleon
+/obj/item/storage/box/syndie_kit/chameleon // No longer contains noslips, but is cheap and has no player minimum
 	name = "chameleon kit"
 
 /obj/item/storage/box/syndie_kit/chameleon/PopulateContents()
 	new /obj/item/clothing/under/chameleon(src)
 	new /obj/item/clothing/suit/chameleon(src)
 	new /obj/item/clothing/gloves/chameleon(src)
-	new /obj/item/clothing/shoes/chameleon(src)

```

Comment:
Make a new subtype `/obj/item/clothing/shoes/chameleon/noslip` and move the noslip flag to that instead imo, that way you don't have to split up the kit like this. :(

Reply:
Yeah, that seems like the proper way compared to having an incomplete chameleon kit.
It's gonna be a while until I can fix my PR, should I leave it open or do people normally close and reopen a new one?